### ISSUE TITLE
Initialization should conform with other adapters

### ISSUE THREAD

**Issue by ekryski** [feathersjs/feathers-knex#24]
All other db adapters support these options:

- Model (required) - The knex object scoped to a table
- id (default: id) [optional] - The name of the id property
- paginate [optional] - A pagination object containing a default and max page size (see below)

So it should look like this:

```js
var knex = require('knex');
var knexService = require('feathers-knex');

app.use('/todos', knexService({
  Model: knex('todos')
}));
```

**Comment by ekryski** [comment]
Closed via 9e7a20cdc9864c6e6115a79f7c42f68819c8a05d

**Comment by ekryski** [comment]
We needed to add the `name` attribute to this adapter because of the way that knex queries work. If you don't make a call to `knex('table_name')` every time then you end up with weird results. An alternative is to use the `.from('table_name')` syntax but you'd still need to call it every time you make a query.
